### ISSUE TITLE
Accessibility labels in Contact list are all "A Contact"

### ISSUE THREAD

**Issue by knguyenBC** [bcgov/bc-wallet-mobile#2008]
### Description of problem
When voice control is enabled, when you enter your Contact list, all contacts are labelled as "A Contact" which doesn't allow a person to tap on a particular Contact with their voice.

### Expected behavior
The accessibility label of each Contact should be the Contact name.

### Steps to reproduce
- enable voice control on iOS
- choose items names as voice control overlay
- go into BC Wallet Contacts List
- try to tap on a particular Contact using only voice

### Environment
- [ ] Occurs on Android
- [x] Occurs on iOS

Build #: 1740

Android Device Model: 

iOS Device Model: iPhone SE

### Workaround
Enable item numbers overlay when configuring voice control

### Severity
- [ ] High
- [x] Medium
- [ ] Low
